pymedphys/tests/dicom/test_anonymise.py

```diff
@@ -169,6 +169,27 @@ def test_anonymised_dataset_with_empty_patient_sex():
     assert hardcode_replace_ds["PatientSex"].value is None
 
 
+@pytest.mark.slow
+@pytest.mark.pydicom
+def test_alternative_identifying_keywords():
+    default_keyword_list = list(IDENTIFYING_KEYWORDS)
+    alternative_keyword_list = list(default_keyword_list)
```

Comment:
This does make a copy, but can we make it more explicit, as in, we're not trying to convert the variable to a list (what `list(object)` is for), but instead the point of this line is to make a copy:

```python
import copy

# ...

alternative_keyword_list = copy.copy(default_keyword_list)
```

Reply:
Sure